### PR TITLE
Add Spanish lang and Auto lang setter

### PR DESCRIPTION
**Pull request by Carlitosnu**
I renamed the name of files and set spanish lang. Then read the header `accept-language` and find the lang

### CODE REVIEW

**Comment by Carlitosnu** [comment]
I'm other time

**Review by mrjvs — COMMENTED** [review]
Besides code comments a few things to note:
1. We are currently reworking the entire site on the dev branch, your pr will have to be remade to fit that branch
2. Language selection shouldn't be done only through accept-language header. I'm not sure if we even want to use the header as it makes things complicated. (i.e. how would you set a preference outside of the header, a dropdown on the browser side would have to somehow set a cookie so the serverside can see it)

File: config.example.json [hunk 1]
```diff
@@ -1,5 +0,0 @@
-{
-	"http": {
-		"port": 8080
-	}
-}
```
[the remark this answers is not in the document]
Comment (Carlitosnu) [reply config.example.json:5]: excuseme

**Review by Carlitosnu — COMMENTED** [review]
Uhmmm sorry, i fix all and re-pull request

**Pull request closed without merging**